data/crac/crac-result-extensions/src/main/java/com/farao_community/farao/data/crac_result_extensions/NetworkActionResult.java

```diff
@@ -7,16 +7,28 @@
 
 package com.farao_community.farao.data.crac_result_extensions;
 
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Set;
 
 /**
  * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
  */
+@JsonTypeName("network-action-result")
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
```

Comment:
does it mean you do not need a getter ?
well done for finding this.

Reply:
yes! thanks to @phiedw for finding this :)